backend/src/acidwatch_api/models/gibbs_minimization_model.py

```diff
@@ -100,38 +166,73 @@ async def run(self) -> RunResult:
         system.addComponent("water", 0.0, "mole/sec")
-        system.addComponent("ammonia", 0.0, "mole/sec")
+        # system.addComponent("ammonia", 0.0, "mole/sec")
         system.addComponent("H2S", 0.0, "mole/sec")
         system.addComponent("oxygen", 0.0, "mole/sec")
         system.addComponent("sulfuric acid", 0.0, "mole/sec")
         system.addComponent("nitric acid", 0.0, "mole/sec")
+        system.addComponent("NH4NO3", 0.0, "mole/sec")
+        system.addComponent("NH4HSO4", 0.0, "mole/sec")
+        system.addComponent("formic acid", 0.0, "mole/sec")
+        system.addComponent("acetic acid", 0.0, "mole/sec")
+        system.addComponent("methanol", 0.0, "mole/sec")
+        system.addComponent("ethanol", 0.0, "mole/sec")
+        system.addComponent("CO", 0.0, "mole/sec")
+        # system.addComponent("hydrogen", 0.0, "mole/sec")
+        # system.addComponent("N2O3", 0.0, "mole/sec")
+        # system.addComponent("N2O", 0.0, "mole/sec")
+        # system.addComponent("nitrogen", 0.0, "mole/sec")
+        system.addComponent("NH2OH", 0.0, "mole/sec")
+        # system.addComponent("N2H4", 0.0, "mole/sec")
         system.addComponent("S8", 0.0, "mole/sec")
-
-        system.setMixingRule(2)
-        system.setMultiPhaseCheck(True)
+        system.addComponent("HNO2", 0.0, "mole/sec")
+        system.addComponent("MEG", 0.0, "mole/sec")
+        system.addComponent("DEG", 0.0, "mole/sec")
+        system.addComponent("TEG", 0.0, "mole/sec")
+        system.addComponent("MEA", 0.0, "mole/sec")
+        system.addComponent("MDEA", 0.0, "mole/sec")
+        system.addComponent("DEA", 0.0, "mole/sec")
+        # system.addComponent("methane", 0.0, "mole/sec")
+        system.addComponent("ethane", 0.0, "mole/sec")
+        system.addComponent("propane", 0.0, "mole/sec")
+        system.addComponent("i-butane", 0.0, "mole/sec")
+        system.addComponent("n-butane", 0.0, "mole/sec")
+        system.addComponent("i-pentane", 0.0, "mole/sec")
+        system.addComponent("n-pentane", 0.0, "mole/sec")
+        system.addComponent("ethylene", 0.0, "mole/sec")
+        system.addComponent("benzene", 0.0, "mole/sec")
+        system.addComponent("toluene", 0.0, "mole/sec")
+        system.addComponent("o-Xylene", 0.0, "mole/sec")
+        system.addComponent("HCN", 0.0, "mole/sec")
+        # system.addComponent("COS", 0.0, "mole/sec")
+        system.addComponent("CS2", 0.0, "mole/sec")
+        system.addComponent("argon", 0.0, "mole/sec")
+        system.addComponent("CH2O", 0.0, "mole/sec")
+        system.addComponent("C2H4O", 0.0, "mole/sec")
+        system.addComponent("C2H4", 0.0, "mole/sec")
 
         if eos in (_EquationOfState.SRK, _EquationOfState.PR):
             system.setMixingRule(2)
         elif eos == _EquationOfState.SRKCPA:
             system.setMixingRule(10)
-        else:
-            raise NotImplementedError(f"Equation of state not implemented: {eos}")
```

Comment:
The reason such a `raise` should be included is because we want to know when things aren't working as expected. It is okay that it fails. It is in fact much much better that it fails when it should, rather than just happily ignoring things. So if we add an `_EquationOfState` in the `enum`, we want to make sure we don't forget to add how that is handled down here. A `failure` at this point should definitively be flagged, cause we have probably forgot something when writing the code...

Reply:
Alright, let's keep it as you made it now then :) 

(fetched the code though, and it seems there's some shifting/duplication of lines that doesn't work out. E.g. the reactor is made twice..)